bot/cogs/pavlov_admin.py

```diff
@@ -231,6 +234,7 @@ async def repeat(
         if not await check_perm_admin(ctx, server_name):
             return
         for i in range(int(aot)):
+            time.sleep(0.2)
```

Comment:
`discord.py` is an asynchronous framework, `time.sleep` is synchronous. This sleep would block the whole bot for the duration, please use `asyncio.sleep` instead

Reply:
Doesn't really make any sense 🤔 what do you mean it doesn't do anything?